Readme update ssh

joywa opened the pull request:
Issue #, if available:

Description of changes:

- Added instructions on how to create a new EC2 instance and set it up in the same VPC as Neptune to be used as a bastion server
- Added instructions for setting up an SSH tunnel on Windows

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

beebs-systap commented on README.md:
```diff
@@ -85,7 +85,47 @@ echo "{
 
 ### Connecting to a local graph store
-As mentioned in the introduction, it is possible to connect `graph-notebook` to a graph database running on your local machine. An example being Gremlin Server. There are additional instructions regarding the use of local servers in the `additional-databases` folder.
+As mentioned in the introduction, it is possible to connect `graph-notebook` to a graph database running on your local machine, an example being Gremlin Server. There are additional instructions regarding the use of local servers in the `additional-databases` folder.
```
Can you make graph-notebook a link to the directory?

beebs-systap replied:
Same for additional-databases.

joywa replied:
Done!

krlawrence commented:
It is probably worth mentioning that an SSH tunnel is only one way to connect to Neptune. It's a simple and convenient way but there are others such as creating a load balancer.

austinkline commented on README.md:
```diff
@@ -85,7 +85,47 @@ echo "{
 
 ### Connecting to a local graph store
-As mentioned in the introduction, it is possible to connect `graph-notebook` to a graph database running on your local machine. An example being Gremlin Server. There are additional instructions regarding the use of local servers in the `additional-databases` folder.
+As mentioned in the introduction, it is possible to connect [`graph-notebook`](src/graph_notebook) to a graph database running on your local machine, an example being Gremlin Server. There are additional instructions regarding the use of local servers in the [`additional-databases`](additional-databases) folder.
```
I would like to keep our primary README smaller, can we move this to a directory like we did for [gremlin-server](https://github.com/aws/graph-notebook/tree/main/additional-databases/gremlin-server)?

austinkline reviewed (changes requested):
I would like to prevent our main README from getting too large, let's keep each type of connection in its own directory. Whatever is in this file is what will be shown by our page in [pypi](https://pypi.org/project/graph-notebook/)

joywa commented:
Added this line to the first paragraph of Neptune README.md: "Please note that this wiki is not an official recommendation on network setups as there are many ways to connect to Amazon Neptune from outside of the VPC, such as setting up a load balancer or VPC peering." @krlawrence 

Moved content to from primary README.md to neptune/README.md @austinkline

austinkline commented on additional-databases/neptune/README.md:
```diff
@@ -0,0 +1,38 @@
+## Connecting a local graph-notebook to Amazon Neptune (first-time setup)
+When using graph-notebook locally to connect to an Amazon Neptune database for the first time, there are a couple of additional steps. This section assumes that you've already installed & configured [graph-notebook](#installation) locally.  Please note that this wiki is not an official recommendation on network setups as there are many ways to connect to Amazon Neptune from outside of the VPC, su ...
```
the hyper-link \[graph-notebook](#installation) likely will not work now since we have moved this to another directory. It should now be \[graph-notebook](https://github.com/aws/graph-notebook#installation)

austinkline commented on additional-databases/neptune/README.md:
```diff
@@ -0,0 +1,38 @@
+
+Make sure the EC2 instance is in the same VPC group as your Neptune cluster. To find the VPC group for your Neptune cluster, check the console under [Neptune](https://console.aws.amazon.com/neptune/home) > Subnet groups. The instance's security group needs to be able to send and receive on port 22 for SSH and port 8182 for Neptune.  See below for an example security group setup.  
+
+![Sample EC2 Inbound Rules](/././images/sample-ec2rules.png)
```
I do not think this image url will work, maybe something like:

\!\[Sample EC2 Inbound Rules](https://github.com/aws/graph-notebook/blob/main/images/sample-ec2rules.png)

austinkline commented on additional-databases/neptune/README.md:
```diff
@@ -0,0 +1,38 @@
+
+You should see a status report indicating the health of your cluster like so:
+
+`{"status":"healthy","startTime":"Wed Nov 04 23:24:44 UTC 2020","dbEngineVersion":"1.0.3.0.R1","role":"writer","gremlin":{"version":"tinkerpop-3.4.3"},"sparql":{"version":"sparql-1.1"},"labMode":{"ObjectIndex":"disabled","DFEQueryEngine":"disabled","ReadWriteConflictDetection":"enabled"}}`
```
nit: can we do the full code-block here instead of the single one?

\```
{
  "status": "healthy",
  "startTime": "Wed Nov 04 23:24:44 UTC 2020",
  "dbEngineVersion": "1.0.3.0.R1",
  "role": "writer",
  "gremlin": {
    "version": "tinkerpop-3.4.3"
  },
  "sparql": {
    "version": "sparql-1.1"
  },
  "labMode": {
    "ObjectIndex": "disabled",
    "DFEQueryEngine": "disabled",
    "ReadWriteConflictDetection": "enabled"
  }
}
\```

krlawrence commented:
The JSON seems to have lost its indentation.

krlawrence commented on additional-databases/neptune/README.md:
````diff
@@ -0,0 +1,56 @@
+You should see a report, similar to the one below, indicating the status and details of your specific cluster:
+
+```
+{
````
Indentation here seems to have been lost
